third_party/vp8_fixed_q.patch

```diff
@@ -51,12 +51,12 @@ index 3cceb5a..c737d3c 100644
          if (oxcf->alt_q < 0)
              cpi->oxcf.alt_q = q_trans[0];
 diff --git a/vp8/vp8_cx_iface.c b/vp8/vp8_cx_iface.c
-index b1b079c..b6bef48 100644
+index 22a82b7..0c75011 100644
 --- a/vp8/vp8_cx_iface.c
 +++ b/vp8/vp8_cx_iface.c
-@@ -132,8 +132,19 @@ static vpx_codec_err_t validate_config(vpx_codec_alg_priv_t      *ctx,
+@@ -139,8 +139,19 @@ static vpx_codec_err_t validate_config(vpx_codec_alg_priv_t      *ctx,
      RANGE_CHECK(cfg, g_timebase.den,        1, 1000000000);
-     RANGE_CHECK(cfg, g_timebase.num,        1, cfg->g_timebase.den);
+     RANGE_CHECK(cfg, g_timebase.num,        1, 1000000000);
```

Comment:
All the changes in vp8_fixed_q.patch are made by:

- Applying the previous version of the patch
- Finding out were the patch failed to apply
- Adjusting the code until the non-applying part worked again

So this shouldn't be a change that I wrote, it should be a change from 1.3 to 1.6 in the file being patched. But I need to read the file really carefully to be sure that's the case!